packages/@ember/-internals/metal/lib/injected_property.ts

```diff
@@ -87,3 +80,5 @@ export default function inject(
     return decorator;
   }
 }
+
+export default inject;
```

Comment:
This is a workaround for the "multiple default exports" issue from `import/export`

This change is in a seperate commit, so is easy to revert..